src/commands/deploy.js

```diff
@@ -29,18 +29,12 @@ class DeployCommand extends AbstractAuthenticatedCommand {
    * Get selected environment; prompt for environment when none is provided.
    * @param {Object} config - Actual command config (including command parameters).
    * @throws Will throw an error when there is no environment (unreachable).
-   * @throws Will throw an error when there is no production environment.
+   * @throws Will throw an error when there is no reference environment.
    * @return {Object} The environment found.
    */
   async getEnvironment(config) {
     const environments = await new EnvironmentManager(config).listEnvironments();
-    const productionExists = environments.find((environment) => environment.type === 'production');
-
-    if (environments.length === 0) throw new Error('❌ No environment found.');
```

Comment:
this line should be kept I think
If the list is empty it's good to stop the action